### DISCUSSION TITLE
How to change a negative trend

### DISCUSSION THREAD

**Discussion by sohanasarah** [ourownstory/neural_prophet#1253] [Q&A - get help using NeuralProphet] [answered]
So I am generating a forecast for a dataset that always shows a negative trend. I removed outliers from the data and tried tuning the parameters. I also tried to clip the forecast result so it doesn't produce negative forecasts. I am not being able to change the downward trend. Do you have any suggestions on how to handle this situation?

![negative trend](https://user-images.githubusercontent.com/38297094/231006486-b7f59f1d-e064-440e-a859-13e02284d4a9.png)

Parameters I'm using:

params = dict(weekly_seasonality = False,
            daily_seasonality = False,
            yearly_seasonality= True,
            seasonality_mode = 'multiplicative',
            epochs=500,
            learning_rate=0.01,
            trend_reg=0.7)

**Comment by sohanasarah** [comment]
I have also added custom changepoints, set trend_reg_threshold to true and newer_samples_weight to 4,

**Reply by judussoari** [reply, marked as answer]
Hi @sohanasarah could you please share about the data you are looking at? I assume the green curve is the input time series, which would mean that you are trying to forecast 5 years into the future, which seems a lot to me. Does the same occur for a smaller forecast scope? 
Also, you have not activated autoregression yet (using `n_lags` as a param). This accounts for the autocorrelation of the time series. I can imagine this to be what you are looking for
